apps/modelviewer/vite.config.ts

```diff
@@ -1,4 +1,18 @@
-import viteConfigBase from '../../vite.config.base';
-export default {
-  ...viteConfigBase,
-};
+import { defineConfig } from 'vite';
+import { InjectProcessPlugin } from '../../patches/3d-patch.ts';
+
+export default defineConfig({
+  appType: 'custom',
+
+  build: {
+    emptyOutDir: true,
+    rollupOptions: {
+      plugins: [InjectProcessPlugin],
```

Comment:
TOOD: investigate if this is actually necessary when the 3d package is bundled as a widget. You would possibly need this in the widget vite config

Reply:
Lets discuss the necessity of this app

Reply:
This app is now removed